### DISCUSSION TITLE
installer quits with "exit status 78" and "Error in network definition: vlan10: interface 'enp0s3' is not defined"

### DISCUSSION THREAD

**Discussion by lacak-sk** [utmstack/UTMStack#602] [Q&A] [answered]
I am installing into VirtualBox 7.0.14 hosted on Windows 10 from ISO installer "utmstack-v10-1-0-autoinstall.iso".
After Ubuntu Linux is installed and VM reboots I log in (as utmstack) and run "sudo ./installer", which after initial checks quits with exit code 78.
Network adapter on VM is in bridged mode and IP address is obtained from our internal DHCP server ("ip a" shows: 10.0.0.20/28, GW 10.0.0.30)
There is in /etc/netplan/ file "00-installer-config.yaml-orig". If I rename this file to "00-installer-config.yaml" then when I again run "sudo ./installer" then installation completes.
Does it indicates any problem?
(after UTMStackServer is installed I fall into another problem #599 ...)

**Comment by c3s4rfred** [comment]
Hi, @lacak-sk you should avoid using those ranges of IP, docker, Virtualbox and other kinds of tools will try to use the first ranges of IP and will cause issues. UTMStack is based on docker, try to install the ubuntu and configure the network with other IP range different from docker's, then install UTMStack.

Best regards

**Comment by lacak-sk** [comment]
Thanks. If we can't use address ranges 10.0.0.x, 10.0.1.y then it's quite a problem because these ranges are quite common in LANs.
Is it not possible to reconfigure docker containers to use address ranges that are not so common (e.g. 10.200.0.0, 10.201.0.0, etc.)? - so that they did not conflict with the address ranges we use in our LAN (otherwise we are not actually able to deploy UTMStack).
VirtualBox itself does not play a role here because it does not use other ranges of IP addresses, but only those that are already in the LAN.

If none is possible then at least it should be documented (that UTMStack can not be deployed on networks which utilize address ranges 10.0.0.0-10.0.?.?) ...

**Comment by c3s4rfred** [comment]
Hi, @lacak-sk we'll analyze the inclusion of your proposal in our documentation. Try to reboot the machine after installation and tell us if solved your issue, sometimes a reboot is needed to refresh the virtual network configuration.

Best regards

**Comment by lacak-sk** [comment]
No, reboot does not solve problem. I rebooted VM many times before I raise questions ...

**Reply by c3s4rfred** [reply]
Hi @lacak-sk, we have released a new version of UTMStack with bug fixes related to the network and resources distribution, please, read the documentation, the system requirements have changed, and upgrade your instance. Let us know the results.

Best regards

**Reply by lacak-sk** [reply]
Unfortunately new installer requires at least 15490 MB of RAM. In my VM I can allocate maximum of 14670 MB (I have 16 GB physical memory). Why are the memory requirement so high? (for testing/evaluating environments with small number of devices and basic integrations 15490 MB is IMO too strong requirement! compared to 11 GB in 10.3.0)

(of course on another VM where I have already installed 10.3.0 before, I can rerun existing ./installer, which upgrades UTMStack 10.3.0 to 10.4.0 ... so it seems, that also version 10.4.0 can run without any problems on VM with 11 GB RAM)

**Reply by osmontero** [reply, marked as answer]
UTMStack requires a minimum of 16GB of RAM for stable operation.  With less memory, the system risks severe instability, potentially leading to data loss, compromised reliability, and failure to generate critical alerts. This is particularly true for the correlation engine, which relies heavily on in-memory data analysis for optimal performance.

**Reply by osmontero** [reply]
If you're comfortable with the significant risks and possess the technical expertise, you can fork the repository, modify the memory requirements, and compile your own installer. Be aware that this could lead to unpredictable behavior and potential instability.
